utils/build_swift/build_swift/driver_arguments.py

```diff
@@ -334,6 +334,9 @@ def create_argument_parser():
            default=defaults.SWIFT_ANALYZE_CODE_COVERAGE,
            help='enable code coverage analysis in Swift (false, not-merged, '
                 'merged).')
+                
+    option('--swift-disable-dead-stripping', toggle_true, 
+           help="Disable dead stripping in Swift build")
```

Comment:
For the help text, I think we should be more specific, say something like "Turn off Darwin-specific dead stripping for swiftc and other tools."

Reply:
Thanks! I changed it to "Turn off Darwin-specific dead stripping for Swift host tools." as you suggested :)